src/main/java/com/kammatysiak/medicalclinic/model/entity/Doctor.java

```diff
@@ -33,6 +32,9 @@ public class Doctor {
     private String password;
     @Column(name = "MODIFY_DATE")
     private LocalDateTime modifyDate;
+    @OneToMany
+    @JoinColumn(name = "VISITS")
```

Comment:
Wydaje mi sie ze jest to tutaj niepotrzebne.
@OneToMany(mappedBy = "doctor") <- doctor bo tak nazywa sie pole w Visit